test.sh

```diff
@@ -165,6 +170,47 @@ do
 				'--header' 'Content-Disposition: attachment; filename=metadata.xml' \
 				'--header' 'Packaging: application/atom+xml;type=entry'
 		fi
+		# Try dynamic updates
+		UPDATEFILE=`dirname $m`/update.xsl
```

Comment:
Where might update.xsl come from?

Reply:
This will be a future file in any `input/*` directory.  If it exists, we'll use it to create a dynamic change to the metadata, rather than always relying on a hardcoded `update.xml` file.